internal/tokens/tifactory/factory.go

```diff
@@ -30,6 +30,16 @@ var pluginsByName = map[string]func() tokens.Plugin{
 	(*fftokens.FFTokens)(nil).Name(): func() tokens.Plugin { return &fftokens.FFTokens{} },
 }
 
+func InitPrefixWithSubPrefix(prefix config.PrefixArray) {
```

Comment:
I think these names are going to change again when we bring in the latest firefly-common... but I think this one should probably land as "InitConfig" and the deprecated one below should be "InitConfigDeprecated", or something like that.

Reply:
Agreed, I'll include these in the PR for upgrading to latest firefly-common